src/strategies/base.ts

```diff
@@ -227,6 +229,8 @@ export abstract class BaseStrategy implements Strategy {
       await this.buildVersionsMap(conventionalCommits),
       conventionalCommits
     );
+    this.packageName = this.packageName ?? (await this.getDefaultPackageName());
+    logger.debug('packageName', this.packageName);
```

Comment:
This is potentially an unnecessary call. We should probably make this a memoizing getter and only call it when necessary.